### PR TITLE
Migrate the iMOD Coupler docs

### PR DESCRIPTION
**Pull request by Hofer-Julian**
For now only metamod is documented

### CODE REVIEW

**Review by JoerivanEngelen — COMMENTED** [review]
I got 2 comments, but looks good overall.

File: docs/coupler_metamod_config.qmd [hunk 1]
```diff
@@ -0,0 +1,168 @@
+---
+title: Configuration
+---
+
+The configuration file is necessary to describe the model and its dependencies. It is in the [toml](https://toml.io/en/) format and should have a `.toml` extension.
+
+Note that `toml` uses quote marks differently than `python`. Single quotes in toml (`''`) are interpreted similarly to how python would interpret a rawstring (`r''` or `r""`), whereas double quotes (`""`) are interpreted in a similar manner to regular strings in python (`""` or `''`). This matters for paths on Windows, for which we advice to use single quotes.
```
[the remark this answers is not in the document]
Comment (Hofer-Julian) [reply docs/coupler_metamod_config.qmd:7]: > If you agree with me

I don't, but I will still do it :P

File: docs/coupler_index.qmd [hunk 2]
```diff
@@ -29,28 +29,8 @@ paper.](https://www.sciencedirect.com/science/article/pii/S1364815221002991)
...
-The developments on `xmipy` can be found on:
+The `xmipy` library can be found on:
 <https://github.com/Deltares/xmipy>
 
-The example of `imod_coupler` with the coupling between MODFLOW 6 and
-the unsaturated zone model MetaSWAP can be found on:
+The iMOD Coupler can be found on:
 <https://github.com/Deltares/imod_coupler>
-
-## Known Issues
```
[the remark this answers is not in the document]
Comment (Hofer-Julian) [reply docs/coupler_index.qmd:39]: I think the issue has been solved for more than 1.5 years now. I thought that's enough time to remove the warning.
I can still keep it if you prefer so.

File: docs/coupler_metamod_config.qmd (hunk 1 above)
[the remark this answers is not in the document]
Comment (Hofer-Julian) [reply docs/coupler_metamod_config.qmd:9]: Done!

File: docs/coupler_index.qmd (hunk 2 above)
[the remark this answers is not in the document]
Comment (Hofer-Julian) [reply docs/coupler_index.qmd:39]: I've added it again

**Pull request merged**